tasks/utils/tokens.ts

```diff
@@ -209,6 +210,31 @@ export const BUSD: Token = {
     vault: "0xD124B55f70D374F58455c8AEdf308E52Cf2A6207",
 }
 
+// NonPeggedFeederPool contains priceGetter
+export const RAI: Token = {
+    symbol: "RAI",
+    address: "0x03ab458634910aad20ef5f1c8ee96f1d6ac54919",
+    chain: Chain.mainnet,
+    platform: Platform.Aave,
+    integrator: "",
```

Comment:
Will need to deploy an integrator and link the address here in order for it to be deposited onto AAVE.